jwst/pipeline/calwebb_detector1.py

```diff
@@ -76,6 +76,13 @@ def process(self, input):
         # open the input as a RampModel
         input = datamodels.RampModel(input)
 
+        if self.ramp_fit.algorithm.upper() == "LIKELY":
+            ngroups = input.data.shape[1]
```

Comment:
Is there a reason to check the shape here instead of the metadata?

Reply:
The actual data shape is what matters, so I check the actual data shape.